.github/workflows/stale_issues_and_pr_cleanup.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Close stale issues and PRs'
+on:
```

Comment:
I wonder if we should also make it so we can start the action manually for debugging?